### PR TITLE
general settings

### PR DESCRIPTION
**Pull request by YuliaSimanavets**

### CODE REVIEW

**Review by vkostechko-applace — COMMENTED** [review]
please change destination branch to develop

File: RingSizer/RingSizer/Base/Extensions/UIView.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import UIKit
+
+extension UIView {
+    func addSubviews(_ views: UIView...) {
+        addSubviews(views)
+    }
+
+    func addSubviews(_ views: [UIView]) {
+        for view in views {
+            addSubview(view)
```
Comment (vkostechko-applace) [inline RingSizer/RingSizer/Base/Extensions/UIView.swift:18]: views.forEach { addSubview($0) }

File: RingSizer/RingSizer/Base/Extensions/UIColor.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+//  UIColor.swift
+//  BooksApp
+//
+//  Created by Yuliya on 03/11/2023.
+//
+
+import Foundation
+import UIKit
+
+public extension UIColor {
```
Comment (vkostechko-applace) [inline RingSizer/RingSizer/Base/Extensions/UIColor.swift:11]: I suggest to use SwiftGen

File: RingSizer/RingSizer/Base/Extensions/UIView.swift [hunk 3]
```diff
@@ -0,0 +1,21 @@
+//
+//  UIView.swift
+//  BooksApp
```
Comment (vkostechko-applace) [inline RingSizer/RingSizer/Base/Extensions/UIView.swift:3]: copy-paste :) 
Not the best file name. 
name it something like "UIView+Utils"

**Review by vkostechko-applace — CHANGES_REQUESTED** [review]

File: RingSizer/RingSizer/Base/Extensions/UIView.swift [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+//
+//  Created by Yuliya on 03/11/2023.
+//
+
+import Foundation
+import UIKit
```
[the remark this answers is not in the document]
Comment (YuliaSimanavets) [reply RingSizer/RingSizer/Base/Extensions/UIView.swift:9]: I got it, thanks

File: RingSizer/RingSizer/Base/Extensions/UIView.swift (hunk 3 above)
Comment (YuliaSimanavets) [reply RingSizer/RingSizer/Base/Extensions/UIView.swift:3]: sorry :)
changed it

File: RingSizer/RingSizer/Base/Extensions/UIColor.swift (hunk 2 above)
Comment (YuliaSimanavets) [reply RingSizer/RingSizer/Base/Extensions/UIColor.swift:11]: got it, I'll do it

**Pull request merged**